list.txt

```diff
@@ -12533,6 +12685,7 @@ gmail-box.com
 gmail-fiji.gq
 gmail.ax
 gmail.co5
+gmail.com
```

Comment:
gmail.com is not a disposable email provider :)

Reply:
Woops mb, i'll recheck and commit :)

Reply:
Done